pkgs/development/python-modules/tlslite-ng/default.nix

```diff
@@ -1,30 +1,37 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
-  pythonAtLeast,
+  fetchFromGitHub,
   ecdsa,
+  hypothesis,
+  pythonAtLeast,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "tlslite-ng";
-  version = "0.7.6";
+  version = "0.8.0b1-unstable-2024-06-24";
   format = "setuptools";
```

Comment:
Can we switch that to pyproject = true?

Reply:
Done, but why?

Reply:
The setuptools format effectively calls `python setup.py bdist_wheel`, but we would like to use `python -m build` instead. If you care about the why, read [PEP518](https://peps.python.org/pep-0517/).